code/modules/materials/materials_ore.dm

```diff
@@ -118,9 +118,9 @@
 /obj/item/stack/material/ore/coal
 	material = /decl/material/solid/graphite
 /obj/item/stack/material/ore/sand
```

Comment:
Does this line up with `default_solid_form` on `/decl/material/solid/sand`? It seems like it might cause a mismatch between stuff spawned directly and stuff spawned via SSmaterials helpers.